packages/react-i18n/src/constants.ts

```diff
@@ -0,0 +1,147 @@
+export enum Weekdays {
+  Sunday = 0,
+  Monday = 1,
+  Tuesday = 2,
+  Wednesday = 3,
+  Thursday = 4,
+  Friday = 5,
+  Saturday = 6,
+}
+
+export const DEFAULT_WEEK_START_DAY = Weekdays.Sunday;
+export const WEEK_START_DAYS = new Map([
+  // Saturday
+  ['AF', Weekdays.Saturday],
+  ['DZ', Weekdays.Saturday],
+  ['BH', Weekdays.Saturday],
+  ['EG', Weekdays.Saturday],
+  ['IR', Weekdays.Saturday],
+  ['IQ', Weekdays.Saturday],
+  ['JO', Weekdays.Saturday],
+  ['KW', Weekdays.Saturday],
+  ['LY', Weekdays.Saturday],
+  ['OM', Weekdays.Saturday],
+  ['QA', Weekdays.Saturday],
+  ['SA', Weekdays.Saturday],
+  ['SY', Weekdays.Saturday],
+  ['AE', Weekdays.Saturday],
+  ['YE', Weekdays.Saturday],
+  // Sunday
```

Comment:
Hmm. Technically we don't but it doesn't feel right. 🤔 
I'm fine removing it though.

Reply:
Actually, there is a problem if we decide to change the `DEFAULT_WEEK_START_DAY`.
Right now it's arbitrarily `Sunday` but it could be Monday. 🤔 I guess I would leave it here after all for now?

Reply:
Sounds good! 👍